src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -397,8 +402,69 @@ def makeSubscriptions(self, workflow):
             pass
 
         elif self.msConfig['defaultDataManSys'] == 'Rucio':
-            pass
+            if isinstance(workflow, MSOutputTemplate):
+                msg = "Making transfer subscriptions for %s"
+                self.logger.info(msg, workflow['RequestName'])
 
+                rucioResultList = []
+                if workflow['destinationOutputMap']:
+                    for dMap in workflow['destinationOutputMap']:
+                        try:
+                            msg = "Making transfer subscriptions for %s"
+                            self.logger.info(msg, workflow['RequestName'])
+                            copies = workflow['numberOfCopies'] if workflow['numberOfCopies'] else 1
+                            # NOTE:
+                            #    Once we get rid of DDM this rseExpression generation
+                            #    should go in the Producer thread
+                            if workflow['isRelVal']:
+                                pnnExpr = '('+'|'.join(["pnn=%s" % pnn for pnn in dMap['destination']])+')'
+                                rseExpression = pnnExpr + '&cms_type=real&rse_type=DISK'
+                            else:
+                                rseExpression = '(tier=2|tier=1)&cms_type=real&rse_type=DISK'
+                            scope = 'cms'
+                            if self.msConfig['enableDataPlacement']:
+                                rucioResultList.append(self.rucio.createReplicationRule(dMap['datasets'],
+                                                                                        rseExpression,
+                                                                                        copies=copies,
+                                                                                        scope=scope))
+                            else:
+                                msg = "Dry run mode. The effective Rucio submission would look like: \n"
```

Comment:
To keep it consistent with the MSTransferor code, perhaps make it big, like `DRY-RUN: blah blah`, e.g.:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/MicroService/Unified/MSCore.py#L72

Reply:
Agreed. Done!